juntagrico/util/depot_list.py

```diff
@@ -17,10 +17,9 @@ def default_depot_list_generation(*args, **options):
             'not the specified day for depot list generation, use --force to override')
         return
 
-    if options['future'] or timezone.now().weekday() in Config.depot_list_generation_days():
+    if options['future']:
```

Comment:
Right.

Either this upgrade instruction is documented and the cron job is adjusted
Or we keep backwards compatibility and add a flag `no-future` to prevent changes of depot for the manual use case.

Any preference?

Reply:
I went with the `no-future` solution and a deprecation warning.